docs/_data/table/common-configuration.csv

```diff
@@ -60,7 +69,7 @@ alluxio.underfs.s3a.secure.http.enabled,false
 alluxio.underfs.s3a.server.side.encryption.enabled,false
 alluxio.underfs.s3a.signer.algorithm,
 alluxio.underfs.s3a.socket.timeout,50sec
-alluxio.version,1.7.0
+alluxio.version,1.7.1-SNAPSHOT
```

Comment:
is this correct for a release branch, or should it be the latest released version in the branch?

Reply:
Good catch, unfortunately I had to build 1.7.1-SNAPSHOT to run the doc generator...